frb_example/with_flutter/rust/src/api.rs

```diff
@@ -96,11 +96,13 @@ pub fn off_topic_memory_test_output_complex_struct(len: i32) -> TreeNode {
 }
 
 pub fn off_topic_deliberately_return_error() -> Result<i32> {
+    #[cfg(not(target_family = "wasm"))]
     std::env::set_var("RUST_BACKTRACE", "1"); // optional, just to see more info...
```

Comment:
Btw is there equivalent thing in wasm mode? i.e. if we want to see backtraces in web

Reply:
It seems that backtraces are available on WASM by default.